crates/compiler/can/tests/test_can.rs

```diff
@@ -1231,6 +1231,54 @@ mod test_can {
         assert_eq!(problems, Vec::new());
     }
 
+    #[test]
```

Comment:
You might consider asserting on the can AST (you could print it back using https://github.com/roc-lang/roc/blob/a6cde11bf4183713e15198477e581b114299f709/crates/compiler/can/src/debug/pretty_print.rs#L1) or adding tests in test_mono to assert the code generation is as expected

Reply:
I moved the tests to `test_mono` because that seemed simpler.